discern.lic

```diff
@@ -0,0 +1,46 @@
+custom_require.call(%w[common common-travel events spellmonitor drinfomon])

```

Comment:
Please add a documentation header so we can put additional information on elanthipedia